app/assets/javascripts/discourse/app/controllers/tags-intersection.js

```diff
@@ -0,0 +1,13 @@
+import TagShowController from "discourse/controllers/tag-show";
+import { queryParams } from "discourse/controllers/discovery-sortable";
+
+export default class TagsIntersectionController extends TagShowController {
+  constructor() {
+    super(...arguments);
+
+    this.set("queryParams", [
+      ...Object.keys(queryParams),
```

Comment:
So `queryParams` is already present in the scope here? Would it work with `this.queryParams` too? If yes maybe we use the `this` version to be more explicit.